docs/en/latest/deployments/ack.md

```diff
@@ -37,7 +37,7 @@ As the data plane of apisix-ingress-controller, [Apache APISIX](http://apisix.ap
 
 ```shell
 cd /path/to/apisix-helm-chart
-helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add apisix https://charts.apiseven.com
```

Comment:
The original content is correct, we do need to add bitnami repo to install etcd.